utils/include/WindowsUtils.cpp

```diff
@@ -0,0 +1,52 @@
+#include "stdafx.h"
+#include "PlatformUtils.h"
+
+#ifdef _WIN64
+
+bool WindowsLibraryInititializator::s_is_started_library = false;
```

Comment:
Rework this class according to our discussion

Reply:
I made it yesterday, but it's on my PC. I deleted this class, added two initialize functions. Now I start the library in CAcceptorWrapper and CConnectorWrapper only once in constructor and finalize it in destructor.